### PR TITLE
dApps competition linking

### PR DESCRIPTION
**Pull request by Woodehh**
We're now able to show the dApp competitions in the footer

### CODE REVIEW

File: content/page/dapps/2.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 url: /dapps/2
 aliases:
 - /dapps/2.html
+- /dapps/2_launch.html
```
Comment (metachris) [inline content/page/dapps/2.html]: why those new url's?

Comment (Woodehh) [reply content/page/dapps/2.html]: To keep it standardized
 - /dapps/1_launch/
 - /dapps/1/
 - /dapps/2_launch/

It used to be just /dapps/2/ for the launch, now we have the launch included. So when the results come in, we do /dapps/2/

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -26,11 +26,22 @@ <h5>Resources</h5>
...
+                    <li><a class="text-muted" href="/dapps/1_launch/">Competition #1</a></li>
+                    <li><a class="text-muted" href="/dapps/1/">Winners Competition #1</a></li>
+                    <li><a class="text-muted" href="/dapps/2_launch/">Competition #2</a></li>
+                </ul>	            
             </div>
             <div class="col-6 col-md">
                 <h5>Help</h5> If you feel you can improve on anything here, just send us a <a href="https://github.com/CityOfZion/cityofzion-website">pull request</a> ❤️
             </div>
+            <div class="col-xl-12">
+				<center>
```
Comment (metachris) [inline layouts/partials/footer.html]: indentation

File: content/page/dapps/2.html (hunk 1 above)
Comment (metachris) [reply content/page/dapps/2.html]: ok

File: layouts/partials/footer.html (hunk 2 above)
Comment (Woodehh) [reply layouts/partials/footer.html]: It's weird, in the code editor is properly indented. I updated the code now, i looks like github is showing too much tabs.

**Pull request merged**